sentry-core/src/test/java/io/sentry/core/SentryExceptionFactoryTest.kt

```diff
@@ -52,6 +52,7 @@ class SentryExceptionFactoryTest {
 
         assertEquals("message", queue.first.value)
         assertEquals("cause", queue.last.value)
+        // TODO: is it right? or it should be the cause(1) and then the exception(2)?
```

Comment:
@bruno-garcia do you know how it should be the right order of exceptions?

Reply:
It's reversed from the logical order cuz python